### PR TITLE
add meetings page

### PR DESCRIPTION
**Pull request by sduthil**

### CODE REVIEW

**Review by fblackburn1 — CHANGES_REQUESTED** [review]
Thank you, it's very nice to see this kind of documantion again :smiley: 

I find the usage of `Wazo Platform` painful to read everywhere
I suggest to only use `Wazo` to make difference with external user
Because we are already in the Wazo Platform documentation
or always use `external` when we talk about Ethen or the *other* users

File: content/uc-doc/administration/meetings.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+---
+title: Meetings
+---
+
+## Purpose
+
+A Meeting allows Wazo Platform users to get in conference with other people that are not using Wazo
+directly. In other words, Wazo Platform users can create meetings and give a Web URL to anybody to
+join them in a Web conference.
```
[the remark this answers is not in the document]
Comment (sduthil) [reply content/uc-doc/administration/meetings.md]: Done, thanks.

File: content/uc-doc/administration/meetings.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+## Prerequisites
+
+To achieve this feature, there are some prerequisites:
+
+* Wazo Platform server 21.14 or above
+* A web application for Wazo users
+* A web application for external users (can be the same as for Wazo users)
```
[the remark this answers is not in the document]
Comment (sduthil) [reply content/uc-doc/administration/meetings.md]: I added a note that "you should build your own apps" with a link to wazo-webrtc-demo.

File: content/uc-doc/administration/meetings.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+  that will be given to external users to contact your server:
+  
+  ```http
+  POST https://wazo.example.com/api/confd/1.1/ingresses/http
+  Wazo-Tenant: 74f8a451-2b38-4b74-bdfe-8137688cc1e5
+  
+  {
+    "uri": "https://wazo.example.com"
+  }
+  ```
```
[the remark this answers is not in the document]
Comment (sduthil) [reply content/uc-doc/administration/meetings.md:33]: No idea if there's a valid syntax highlighter for this, all I know is that this code block is HTTP protocol :)

**Review by fblackburn1 — DISMISSED** [review]

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**